src/widgets/BuyNftWidget/subcomponents/BuyNftWidgetHeader/BuyNftWidgetHeader.scss

```diff
@@ -0,0 +1,25 @@
+@import "src/styles";
+
+.buy-nft-widget-header {
+    display: flex;
+    justify-content: space-between;
+
+    &__title {
+        font-size: 1.125rem;
+
+        @include for-size(tablet-portrait-up) {
+            font-size: 1.25rem;
+        }
+    }
+
+    &__back-button {
+        display: none;
+
+        @include for-size(tablet-portrait-up) {
+            display: flex;
+            width: 1.375rem;
+            height: 1.375rem;
+            color: var(--c-light-grey);
+        }
+    }
+}
```

Comment:
![image](https://user-images.githubusercontent.com/12832288/219375419-b451ff59-f01a-453b-be7c-f2ebe94442fa.png)
I've seen that once throughout the entire review.. so it's inconsistent.